### PR TITLE
[INFINITY-2293] Document how to use binary secrets in SDK-based services

### PR DESCRIPTION
**Pull request by mbdcos**

### CODE REVIEW

File: docs/pages/operations-guide.md [hunk 1]
```diff
@@ -345,6 +340,23 @@ The path of a secret defines which service IDs can have access to it. You can th
 
 **Note:** Absolute paths (paths with a leading slash) to secrets are not supported. The file path for a secret must be relative to the sandbox.
 
+### Binary Secrets
+
+When you need to store binary files into DC/OS secrets store, for example a Kerberos keytab file, your file needs to be Base64 encoded as specified in RFC 4648. You can use standard `base64` utility.
```
Comment (karya0) [inline docs/pages/operations-guide.md]: s/Base64 encoded/Base64-encoded/

File: docs/pages/operations-guide.md [hunk 2]
```diff
@@ -345,6 +340,23 @@ The path of a secret defines which service IDs can have access to it. You can th
...
+
+``` 
+$  base64 -i krb5.keytab -o kerb5.keytab.base64-encoded
```
Comment (karya0) [inline docs/pages/operations-guide.md]: `-o <filename>` is not available on Linux :(.

Also note that on Linux, `base64` inserts line-feeds in the encoded data and that is not RFC4648-compliant. The fix there is to use `-w 0` to disable line-wrapping. (I am not sure what happens on MAC.) We should explicitly mention the same here.

Comment (mbdcos) [reply docs/pages/operations-guide.md]: Default is `0` in BSD (no line-wrapping).   Parameter is `-b` not -w.  `-o` is available in BSD (Mac).  Linux is stdout only.  Interestingly  BSD, Linux, FreeBSD all different. That is why I specifically mentioned RFC 4648 :)

File: docs/pages/operations-guide.md [hunk 3]
```diff
@@ -345,6 +340,32 @@ The path of a secret defines which service IDs can have access to it. You can th
...
+``` 
+$  base64 -i krb5.keytab -o kerb5.keytab.base64-encoded 
+```
+
+`base64` command line utility in Linux inserts line-feeds in the encoded data by default. Disable line-wrapping via  `-w 0` argument.  Here is a sample base64 command in Linux.
+
+``` 
+$  base64 -w 0 -i krb5.keytab > kerb5.keytab.base64-encoded 
+```
+
```
Comment (karya0) [inline docs/pages/operations-guide.md]: Too many new lines?

**Pull request merged**